### PR TITLE
fix(sab): check for sab before using it

### PR DESCRIPTION
**Pull request by sedghi**
### Context

SAB requires a secure context, and stack viewport can work without it (although MPR still needs a secure context https and CORS headers to work), but this will solve making the viewer actually work in the stack viewport at least.

https://community.ohif.org/t/dicom-images-infinite-loading/831/2?u=jbocce

### Changes & Results

Check for SAB before using it 

### Testing

Run viewer against an unsecure context (localhost is secure by design), run it against an ip

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

- [] I have run the `yarn build:update-api` to update the API documentation, and have
  committed the changes to this PR. (Read more here https://www.cornerstonejs.org/docs/contribute/update-api)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] "OS: 
- [] "Node version: 
- [] "Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by sedghi** [comment]
CC @Ouwen I think this is what you were talking in the office hours

File: packages/dicomImageLoader/src/imageLoader/createImage.ts [hunk 1]
```diff
@@ -120,6 +120,12 @@ function createImage(
     }
   }
 
+  // we need to identify if the target buffer is a SharedArrayBuffer
+  // since inside the webworker we don't have access to the window
+  // to say if it is a SharedArrayBuffer or not with instanceof
+  options.isSharedArrayBuffer =
```
Comment (jbocce) [inline packages/dicomImageLoader/src/imageLoader/createImage.ts:126]: this is still not correct. it should be...

```
  options.isSharedArrayBuffer =
    options.targetBuffer.arrayBuffer &&
    options.targetBuffer.arrayBuffer instanceof SharedArrayBuffer;
```

**Review by jbocce — CHANGES_REQUESTED** [review]
almost there...

**Review by jbocce — APPROVED** [review]
Looks great. Approved.

**Pull request merged**